core/templates/pages/skill-editor-page/navbar/skill-editor-navbar-breadcrumb.component.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for the navbar breadcrumb of the skill editor.
+ */
+
+import { Component } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+import { SkillEditorStateService } from 'pages/skill-editor-page/services/skill-editor-state.service';
+
+@Component({
+  selector: 'skill-editor-navbar-breadcrumb',
```

Comment:
oppia-skill-editor-navbar-breadcrumb*

Reply:
I don't see this pattern anywhere else in the codebase (except for very few components). I think it is ok to leave it this way. What's your thought?

Reply:
We are enforcing this on all the migrated components as per https://angular.io/guide/styleguide#component-custom-prefix

Reply:
Done!